hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/helpers/ChunkUtils.java

```diff
@@ -59,6 +59,9 @@
 
   private static final Set<Path> LOCKS = ConcurrentHashMap.newKeySet();
 
+  private static final Logger LOG =
+      LoggerFactory.getLogger(ChunkManagerImpl.class);
```

Comment:
Shouldn't this be a logger for `ChunkUtils`?  Otherwise logged info will still be confusing, but a bit more efficient.

Reply:
:scream: Ups, thanks. That was the whole point to use the ChunkUtils instead of ChunkManagerImpl. (I just did a wrong merge while I cherry-picked from my dev branch.)